browser/help/topics/faf.md

```diff
@@ -7,10 +7,10 @@ This annotation can be used for filtering variants by allele frequency against a
 
 This calculation only includes non-bottlenecked genetic ancestry groups: we did not calculate this metric on the Amish (`ami`), Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining Individuals (`rmi`) groups.
 
-On the browser, this annotation is directly available on the variant page. In the VCF and Hail Tables, this annotation is computed globally and for each genetic ancestry group. Genetic ancestry group specific AFs are listed separately with 95% and 99% CIs.
+On the browser, this annotation is directly available on the variant page. In the VCF and Hail Tables, this annotation is computed globally and for each genetic ancestry group. Genetic ancestry group specific filtering allele frequencies (FAFs) are listed separately with 95% and 99% CIs.
 
-### <a name="grpmax"></a>Group maximum allele frequency
+### <a name="grpmax"></a>Group maximum filtering allele frequency
 
-This annotation (abbreviated "`grpmax`") contains allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry groups with the highest frequency.
+This annotation (abbreviated "`grpmax FAF`") contains filtering allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry groups with the highest FAF. Note that this field contains filtering allele frequency information from the genetic ancestry group with the highest **FAF**, not the filtering allele frequency information calculated on the genetic ancestry group with the highest **AF**.
```

Comment:
do we actually provide AC, AN, AF, homozygote count for the grpmax FAF? In the tables it looks like just faf95/99_max and faf95/99_max_gen_anc

Reply:
oh, good catch! this text is old and probably describes the grpmax struct

Reply:
@klaricch @ch-kr , I was looking at the v4 genomes tables earlier and trying to figure out the precise difference between the `grpmax` and `fafmax` values. It looks like `fafmax.faf95_max` matches what it on the browser UI labeled `Grpmax Filtering AF More information(95% confidence)`

And `grpmax.AF` matches the `Allele Frequency` (non-filtering) row on the browser UI corresponding to `grpmax.gen_anc`.

But this note on the browser UI calls the "Grpmax Filtering AF" just `grpmax` (called `fafmax` in the hail table), so I think there is some inconsistency between the term `grpmax` on the browser (in this note) and in the hail table and in the hail table documentation here: https://gnomad.broadinstitute.org/help/v4-hts#annotation-descriptions

Reply:
hmm, it looks like one of the changes in this PR didn't get applied:
```
The filtering allele frequency calculation only includes non-bottlenecked genetic ancestry groups: we did not calculate this metric on the Amish (`ami`), Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining Individuals" (`rmi`) groups.

On the browser, this annotation is directly available on the variant page. In the VCF and Hail Tables, this annotation (abbreviated "`faf`") is computed globally and for each genetic ancestry group. Filtering allele frequencies (FAFs) for each genetic ancestry group specific are listed separately with 95% and 99% CIs.
```
![image](https://github.com/broadinstitute/gnomad-browser/assets/19142091/719280e0-50e1-4d5b-9bde-4585217ace6a)


The text in the browser UI (this section):
![image](https://github.com/broadinstitute/gnomad-browser/assets/19142091/de7772ab-2b09-421b-b1fc-1b7e231cf689)
should show that the text describing the filtering allele frequency applies to the `fafmax` annotation in the VCF and Hail Tables. The section below
![image](https://github.com/broadinstitute/gnomad-browser/assets/19142091/8cae1aa6-a0db-4b6f-800b-9cdd004fc042)
now describes `grpmax`, which is distinct from `fafmax`.

I'd welcome any updates if you have suggestions for how to make this text clearer!

Reply:
(cc @rileyhgrant)

Reply:
@ch-kr Ah I see maybe I was looking at an out of date version of the code. It looks like maybe the browser has been partially updated with these changes but not completely. 

One thing that could be improved is to make it totally clear that the field `Grpmax Filtering AF` that this tooltip is displayed on in the browser, is distinct from `Group maximum allele frequency` (also described in this tooltip) in that one is `filtering` and one is not.

It seems like there are 4 concepts:
1) AF: allele frequency (raw, overall + per-group)
2) grpmax AF: grpmax allele frequency (raw, only top group)
3) FAF: filtering allele frequency (filtered, overall + per-group)
4) grpmax FAF: grpmax filtering allele frequency (filtered, only top group)

The gnomad browser at the top shows 1 (AF) and 4 (grpmax FAF), and 2 (grpmax AF) can be looked up in the `Genetic Ancestry Group Frequencies` table on the page. And 3 is not available on the browser. 

And the hail tables call 1 -> `AF/freq`, 2 -> `grpmax`, 3 -> `faf`, and 4 -> `fafmax`.